_data/other.yml

```diff
@@ -547,6 +547,15 @@ websites:
     facebook: spare5app
     email_address: feedback@spare5.com
 
+  - name: TurboSquid
+    url: https://www.turbosquid.com
+    img: turbosquid.png
+    tfa:
+      - email
+      - sms
+    exception: "2FA is only available with a verified email address."
+    doc: https://www.turbosquid.com/MemberInfo/Index.cfm/MFAInfo
```

Comment:
```suggestion
```

I didn't notice that you already provided screenshots on the associated issue. :man_facepalming:

Since the documentation page is not publicly available, there is no benefit in including it here.